### ISSUE TITLE
Front matter is missing in some of the .md files and these files do not get listed in the documentation side menu

### ISSUE THREAD

**Issue by Rajakavitha1** [kubernetes/website#13816]
**This is a Bug Report**

**Problem:**

In continuation with the observations that I made when reviewing https://github.com/kubernetes/website/pull/13782. Here are some of the issues.
1. This line is present in kubefed_init.md, kubefed_join.md, kubefed_options.md, kubefed_unjoin.md, kubefed_version.md files as well.
2. The links to above files in the See Also section results in a 404.
3. The links to these files are missing in the documentation side menu because the front matter is missing in these files. Please see, https://kubernetes.io/docs/contribute/style/content-organization/#documentation-side-menu
4 To maintain consistency please change SEE ALSO to What's next. See, https://kubernetes.io/docs/contribute/style/page-templates/

**Proposed Solution:**
- Update the Front matter in the .md files. Or may be even automate PR check for Front Matter precedent.
-Remove the line that was introduced inadvertently in the files listed above.
-Change SEE ALSO to What's next for consistency and compliance.

**Pages to Update:**
https://kubernetes.io/docs/reference/setup-tools/kubefed/kubefed_init/...
https://kubernetes.io/docs/reference/setup-tools/kubefed/kubefed_join/
https://kubernetes.io/docs/reference/setup-tools/kubefed/kubefed_options/
https://kubernetes.io/docs/reference/setup-tools/kubefed/kubefed_unjoin/
https://kubernetes.io/docs/reference/setup-tools/kubefed/kubefed_version/

<v1.14>

To avoid these issues we could automate PR check for Front Matter precedent.

**Comment by rajeshdeshpande02** [comment]
Thanks @Rajakavitha1 for reviewing PR #13782 which you mentioned in this issue. Will work on these issues as well.

**Comment by kbhawkey** [comment]
Note:
- Manually fixing the pages now is a good idea 😄 
- There is a docs template that was created especially for rendering the tools references. This template may be useful to apply to these pages as the headings follow the other tool refs. The tool ref pages do not require a number of sections required by concept pages. See federation-apiserver.md as an example of a file that uses this template,
```yaml
---
title: federation-apiserver
content_template: templates/tool-reference
weight: 70
---
```
- The setup-tools/kubefed markdown files are generated as part of the docs release. Ideally, the generation scripts could be updated for the 1.15 release.
- A PR was added in kubernetes-incubator/reference-docs to update the reference scripts with call outs to this template. Can't recall whether the script(s) apply to generation of kubefed pages. I excluded kubeadm pages, as these pages are rendered with includes, however, the script could be modified to handle all tool ref files.
